scripts/prepare_sector_network.py

```diff
@@ -4925,7 +5076,7 @@ def add_industry(
         factor = (
             1
             - industrial_demand.loc[loads_i, "current electricity"].sum()
-            / n.loads_t.p_set[loads_i].sum().sum()
+            / n.loads_t.p_set[loads_i].multiply(n.snapshot_weightings.generators, axis=0).sum().sum()
```

Comment:
in prepared_network, the loads shouldn't be multiplied with the snapshot weightings. This is done automatically during solve. If you do it here, it's accounted twice